staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -1613,6 +1613,11 @@ type VirtualMachineStartFailure struct {
 	RetryAfterTimestamp  *metav1.Time `json:"retryAfterTimestamp,omitempty"`
 }
 
+type VolumeMigration struct {
+	// +listType=atomic
+	Volumes []Volume `json:"volumes,omitempty"`
```

Comment:
I assume these are the "old" volumes?

Reply:
I think it would be better if VolumeMigration had another name to reflect that these are `currently` used volumes - or last good known volumes.

Reply:
Ah so the list changes when we cut over to the "new" volumes?